desktop-app/app/containers/Browser/index.js

```diff
@@ -63,6 +64,7 @@ const Browser = ({browser}) => (
       </div>
     </div>
     <AppNotification />
+    <LinkHoverDisplayContainer />
```

Comment:
@rishichawda Can you please move this to the `DevicesPreviewer` component so that this info just stays within the page preview area? 

![97111563-ca26f180-16df-11eb-928e-b672640f0eda](https://user-images.githubusercontent.com/1283424/97254458-67933a00-1834-11eb-97b7-7a31919016f4.png)

Reply:
Yup this seems to be a better way. Thanks, I'll update this. 👍